README.md

```diff
@@ -1 +1,31 @@
-# Awesome-Books
\ No newline at end of file
+# Awesome-Books
+
+This project creates a book collection webpage where one keys in the book title and author.
+
+# Project Requirements
+
+- Create a collection that keeps a list of books (hint: you can use an array of objects for that).
+- Create a function to add a new book to the collection, with title and author.
+- Create a function to remove a book from the collection (hint: you can use the array filter() method).
+- Display all books saved in the collection in the top part of the page.
+- Make sure that when a user clicks the "Add" button:
+- A new book is added to the collection.
+- The new book is displayed in the page.
+- Make sure that when a user clicks the "Remove" button:
+- The correct book is removed from the collection.
+- The correct book dissapears from the page.
+- Make sure that data is preserved in the browser's memory by using localStorage.
+
+## Built With
+
+- HTML
+- CSS
+- Git & Gitflow
+- Javascript
+
+## Joy Kwamboka
+
+- Github: [@kwambiee](https://github.com/kwambiee)
+- Twitter: [@kwambiee](https://twitter.com/kwambiee)
+- Linkedin: [Joy Kwamboka](https://www.linkedin.com/in/joy-kwamboka)
+- E-mail: kwambokaj2.jk@gmail.com
```

Comment:
- [ ] Your readme is missing several sections from the required [Microverse Readme Template](https://github.com/microverseinc/readme-template).

Please, kindly implement the Microverse Readme Template to your project.  Kindly remember that all the sections from the template are required, except for "Live demo" and "Tests" if not required by the project.